docs/ref/forms/widgets.txt

```diff
@@ -718,46 +718,58 @@ that specifies the template used to render each choice. For example, for the
 
     .. code-block:: html+django
 
+        <fieldset>
+        <legend>{{ myform.beatles.label|safe }}</legend>
```

Comment:
Here and below – accessibility-wise this is looking great, but I don’t get why `|safe` would be needed here?

Reply:
You're right, I don't think we need this. I re-tested setting something like "Ben & Jerry" as the label and it rendered just fine.